spynnaker_integration_tests/test_master_pop/spynnaker.cfg

```diff
@@ -1,8 +1,2 @@
-[Reports]
-extract_iobuf = True
-extract_iobuf_from_cores = 0,0,2
-extract_iobuf_from_binary_types = IF_curr_exp.aplx,reverse_iptag_multicast_source.aplx
-clear_iobuf_during_run = True
-
 [Mapping]
```

Comment:
This change doesn't appear to be related to neuromodulation but I agree it should probably be done!

Reply:
I think it was a failing test though I can't quite remember what caused it...  Possibly the fact that one of the binaries wasn't running any more.  The test seemed to be unrelated to the config though...